src/server.c

```diff
@@ -3109,8 +3110,9 @@ void resetCommandTableStats(dict* commands) {
 }
 
 void resetErrorTableStats(void) {
-    raxFreeWithCallback(server.errors, zfree);
+    freeErrorsRadixTreeAsync(server.errors);
     server.errors = raxNew();
+    server.errors_enabled = 1;
```

Comment:
maybe we should drop this line.
it means someone can pollute the log with many lines by a combination of Lua and RESETSTAT

Reply:
server restart.

Reply:
i am ok with that but server restart is a heavy operation. considering config resetstat is a kind of admin-like command, maybe we can keep this line?

Reply:
well, considering that scripts can do `redis.log`, i suppose it's not a real risk.